package.json

```diff
@@ -16,10 +16,12 @@
   "scripts": {
     "build": "npm run storybook:build",
     "build:hooksIndex": "node scripts/createIndexFile.js ./packages/hooks/src",
+    "bootstrap": "npm run bootstrap --workspaces --if-present",
```

Comment:
I have a naive question. What is "bootstrap" actually? We're running `npm run bootstrap` which makes me think it is a script, but then the `bootstrap` script also calls `npm run bootstrap`... ??? 
- signed recursively confused

Reply:
Ah, I found it!
`bootstrap` is a script for each of the underlying "workspaces" which we are triggering here. Ok, I get it. 
(https://docs.npmjs.com/cli/v7/using-npm/workspaces#ignoring-missing-scripts)

Reply:
You got it! I tried to briefly describe that in the PR description but didn't go too in depth.